goreman_test.go

```diff
@@ -0,0 +1,57 @@
+package main
```

Comment:
Thanks. please add build constraint to avoid windows since Windows does not have sleep.exe, I'll add tests on Windows.

Reply:
Oh ok!

Reply:
Done